test/e2e/package.json

```diff
@@ -16,6 +16,7 @@
     "cypress": "^11.2.0",
     "dayjs": "^1.10.4",
     "got": "^11.8.2",
+    "json-schema-to-typescript": "^12.0.0",
```

Comment:
This tool is pretty cool! We now have a generated `d.ts` file of our config, that we can use in the tests to get typesafe config updates. Always bugged me, that I wasn't able to get code completion when working with it.

https://github.com/bcherny/json-schema-to-typescript#readme

Reply:
cool idea